flang/test/Driver/fixed-free-detection.f90

```diff
@@ -25,5 +25,5 @@
 ! MULTIPLEFORMS:program freeform
 ! MULTIPLEFORMS-NOT:programfixedform
 ! MULTIPLEFORMS-NEXT:end
-! MULTIPLEFORMS-NEXT:programfixedform
+! MULTIPLEFORMS:programfixedform
```

Comment:
Why this change?

Reply:
This was done in order to restore the code back to default. This is due to the frame pointer value being specified as argument instead of modifying the test.

Reply:
This change was made when the frame-pointer had as the default value None, therefore setting it to None, impacted the test. How, I can not recall, but since that is no longer the implementation I will remove any changes in this file, thanks for pointing it out.